controllers/image_generator.go

```diff
@@ -213,9 +204,19 @@ func (r *ImageGenerator) createJobFromFile(jobFileName string) (*batchv1.Job, er
 		return nil, err
 	}
 
+	// If PODVM_BUILDER_IMAGE environment variable is set, use it
```

Comment:
PODVM_BUILDER_IMAGE or RELATED_IMAGE_PODVM_BUILDER?

Reply:
thanks for the catch